src/main/res/layout/seek_bar_preference.xml

```diff
@@ -1,47 +1,75 @@
-        android:layout_height="wrap_content"
-        android:layout_toLeftOf="@id/seekBarPrefUnitsRight"
-        android:gravity="right" 
-        />
-    <TextView
-        android:id="@+id/seekBarPrefUnitsLeft"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_toLeftOf="@id/seekBarPrefValue"
-        />
-    <LinearLayout
-        android:id="@+id/seekBarPrefBarContainer"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentLeft="true"
-        android:layout_below="@+id/seekBarPrefUnitsRight"
-        android:orientation="horizontal" 
+        android:layout_height="match_parent"
+        android:layout_marginLeft="15dp"
+        android:layout_marginRight="6dp"
+        android:layout_marginTop="6dp"
+        android:layout_marginBottom="6dp"
+        android:gravity="center_vertical"
         >
-        <SeekBar
-            android:id="@+id/seekBarPrefSeekBar"
+        <TextView
+            android:id="@android:id/title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_toStartOf="@+id/seekBarPrefValue"
+            android:singleLine="true"
+            android:textAppearance="?android:attr/textAppearanceLarge"
+            android:textStyle="bold"
+            android:ellipsize="marquee"
+            android:fadingEdge="horizontal"
+            android:layout_alignParentTop="true"
+            />
+        <TextView
+            android:id="@android:id/summary"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@android:id/title"
+            android:layout_toStartOf="@+id/seekBarPrefValue"
+            android:textAppearance="?android:attr/textAppearanceSmall"
+            android:maxLines="4"
+            />
+        <LinearLayout
+            android:id="@+id/seekBarPrefBarContainer"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:orientation="horizontal" 
+            android:layout_below="@android:id/summary"
+            android:layout_alignParentBottom="true"
+            android:paddingTop="6dp"
+            >
+            <SeekBar
+                android:id="@+id/seekBarPrefSeekBar"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                />
+        </LinearLayout>
+        
+        <TextView
+            android:id="@+id/seekBarPrefUnitsRight"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignBottom="@id/seekBarPrefBarContainer"
+            android:layout_alignParentEnd="true"
+            android:layout_marginBottom="20dp"
```

Comment:
Remember that padding on the seekbar? Well, it looks better if the value isn't so far away. So instead of aligning at the top of the seekbar (above the padding), we align below the seekbar and then use a large margin to push it up to the right height.